app/assets/javascripts/app/responsive_table.js.coffee

```diff
@@ -20,13 +20,13 @@ class window.ResponsiveTable
     @table.find("tbody tr").each (index, row) => @add_header_to_row($(row))
 
   add_header_to_row: ($row) ->
-    $row.find("td").prepend (index) => @responsive_header(index)
+    $row.find("> td").prepend (index) => @responsive_header($row, index)
 
```

Comment:
In this case, I might consider either pulling `$row.find("> td")` into a method with a meaningful name or a comment explaining what `>` does here because I think it's a little opaque otherwise. (Possibly just me not remembering all the jQuery sigils....)